src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/CommunityAPIClient.kt

```diff
@@ -25,7 +25,19 @@ class CommunityAPIClient(
       .subscribe()
   }
 
-  fun <T : Any> makeSyncPostRequest(uri: String, requestBody: Any, responseBodyClass: Class<T>): T? {
+  fun makeAsyncPatchRequest(uri: String, requestBody: Any) {
+    communityApiWebClient.patch().uri(uri)
+      .body(Mono.just(requestBody), requestBody::class.java)
+      .retrieve()
+      .bodyToMono(Unit::class.java)
```

Comment:
what does this line do?

Reply:
i see it above as well, but just want to remind myself

Reply:
My reasoning is that since the response type is Unit, then the response will not be decoded. One is provided by community-api and nothing is done with it.

makeSyncPostRequest has a bodyToMono(AppointmentCreateResponseDTO::class.java) which successfully decodes the response from which we get the delius_appointment_id)